### ISSUE TITLE
Term Request: Vesicular lesions of corneal endothelium

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11751]
**Preferred term label:**
Vesicular lesions of corneal endothelium

**Synonyms**
* Endothelial vesicular opacities
* Crater-like lesions of Descemet membrane
* Donut-like endothelial lesions

**Definition (free text, please give PubMed ID)**
Round, vesicular, or crater-like lesions at the level of the endothelium and Descemet membrane, presenting with specular reflection or retroillumination as dark circular cavities surrounded by a grayish-white halo or halo-like ring, representing foci of abnormal endothelial metaplasia (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Abnormal corneal endothelium morphology (HP:0011488)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Posterior polymorphous corneal dystrophy (PPCD) (ORPHA:98973; PPCD1 OMIM:122000, PPCD3 OMIM:609141, PPCD4 OMIM:618031)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
